### PR TITLE
chore-#181339557-setup unit testing

### PR DESCRIPTION
**Pull request by ericmaniraguha**
# What does this PR do?
Testing 

# Description of Task to be completed?

- [x] configuration of mocha package
- [x] configuration of chai-http package
- [x] configuration of chai package
- [x] configuration of nyc package
- [x]  Tests of welcome.controller.js

# How should this be manually tested?

## How Has This Been Tested?
by using the command : yarn test

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Any background context you want to provide?
None

# What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2555282/stories/181339557

# Screenshots (if appropriate)
Screenshot from 2022-03-02 09-13-15

# Questions:
N/A

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-DB_URL_DEV=postgres://postgres:Password1!@127.0.0.1:5432/dev_barefoot_db
-DB_URL_TEST=postgres://postgres:Password1!@127.0.0.1:5432/test_barefoot_db
-DB_URL_PROD=postgres://postgres:Password1!@127.0.0.1:5432/prod_barefoot_db
+DB_URL_DEV=postgres://orane:andela@127.0.0.1:5432/dev_barefoot_db
+DB_URL_TEST=postgres://orane:andela@127.0.0.1:5432/test_barefoot_db
```
Comment (CYIMANA-Faisal) [inline .env]: We don't have to push the environment variables

File: .nyc_output/3f4f3b80-c9a6-465e-abb5-47af6f0ec59e.json [hunk 2]
```diff
@@ -0,0 +1 @@
...\t\thost: process.env.DB_HOST_DEV,\n\t\tport: process.env.DB_PORT_DEV,\n\t\tdialect: \"postgres\",\n\t\tdialectOptions: {},\n\t\tseederStorage: \"sequelize\",\n\t\tseederStorageTableName: \"sequelize_data\",\n\t},\n\ttest: {\n\t\tuse_env_variable: \"DB_URL_TEST\",\n\t\tusername: process.env.DB_USERNAME_TEST,\n\t\tpassword: process.env.DB_PASSWORD_TEST,\n\t\tdatabase: process.env.DB_DATABASE_TEST,\n\t\thost: process.env.DB_HOST_TEST,\n\t\tport: process.env.DB_PORT_TEST,\n\t\tdialect: \"postgres\",\n\t\tdialectOptions: {},\n\t\tseederStorage: \"sequelize\",\n\t\tseederStorageTableName: \"sequelize_data\",\n\t},\n\tproduction: {\n\t\tuse_env_variable: \"DB_URL_PROD\",\n\t\tusername: process.env.DB_USERNAME_PROD,\n\t\tpassword: process.env.DB_PASSWORD_PROD,\n\t\tdatabase: process.env.DB_DATABASE_PROD,\n\t\thost: process.env.DB_HOST_PROD,\n\t\tport: process.env.DB_PORT_PROD,\n\t\tdialectOptions: {},\n\t\tseederStorage: \"sequelize\",\n\t\tseederStorageTableName: \"sequelize_data\",\n\t},\n};\n"]},"_coverageSchema":"1a1c01bbd47fc00a2c39e90264f33305004495a9","hash":"65eedc9104792e3e7d33b872345f66acc60e0f25","contentHash":"0522866de99206a6bdeed7ac1d3a1e16a5c1c119de9e45d50987de17f306adb3"}}
```
Comment (CYIMANA-Faisal) [inline .nyc_output/3f4f3b80-c9a6-465e-abb5-47af6f0ec59e.json]: please stop pushing the coverage

File: README.md [hunk 3]
```diff
@@ -23,6 +23,12 @@ $ yarn start
 ```
 docs/swagger-ui/
 ```
+## Test configurations packages
```
Comment (CYIMANA-Faisal) [inline README.md]: Basically do yarn install

File: README.md [hunk 4]
```diff
@@ -23,6 +23,12 @@ $ yarn start
...
 docs/swagger-ui/
 ```
+## Test configurations packages 
+
+$ yarn add mocha
+$ yarn add nyc
+$ yarn add chai
+$ yarn add chai-http
 
 ## Contributors
```
Comment (CYIMANA-Faisal) [inline README.md]: Remember to add your self as a contibutor

File: coverage/base.css [hunk 5]
```diff
@@ -0,0 +1,224 @@
+body, html {
```
Comment (CYIMANA-Faisal) [inline coverage/base.css]: No need of this file on git hub repo

**Review by CYIMANA-Faisal — CHANGES_REQUESTED** [review]
@ericmaniraguha Please you don't have to push the coverages and the nyc output foler plus the env file please check all the comments and work on the.I can't forget to mention that you have done a great job

File: .nyc_output/3f4f3b80-c9a6-465e-abb5-47af6f0ec59e.json (hunk 2 above)
Comment (ericmaniraguha) [reply .nyc_output/3f4f3b80-c9a6-465e-abb5-47af6f0ec59e.json]: Noted. Thank you!

File: README.md (hunk 3 above)
Comment (ericmaniraguha) [reply README.md]: Noted. Thank you!

File: coverage/base.css (hunk 5 above)
Comment (ericmaniraguha) [reply coverage/base.css]: Noted. Thank you!

**Pull request merged**